index.ts

```diff
@@ -1,54 +1,31 @@
 import { multiUrlGenerator } from './lib/multiUrlGenerator';
-import readline from 'readline';
+import fetch from 'node-fetch';
 
-const clientUI = readline.createInterface({
-  input: process.stdin,
-  output: process.stdout,
-});
+// Test fetching Wiki articles
+const requestUrls = multiUrlGenerator(
+  'https://$lang.wikipedia.org/wiki/$article',
+  [
+    { $lang: 'en', $article: 'Plato' },
+    { $lang: 'es', $article: 'Aristóteles' },
+  ]
+);
 
-function createObject(
-  home: string,
-  name: string
-): { $home: string; $name: string } {
-  return {
-    $home: home,
-    $name: name,
-  };
-}
-
-function arrayOfObjectsFactory(wnp: string[]) {
-  //The left side of this array contains the names of the websites and the right side contains the names of the paths.
-  // @ts-ignore
-  const arrayOfObjects: { $home: string; $name: string }[] = [].fill(null);
-  wnp.forEach(function (item: string, index: number) {
-    if (index < wnp.length / 2) {
-      arrayOfObjects.push(
-        createObject(wnp[index], wnp[wnp.length / 2 + index])
+async function main() {
+  for (const url of requestUrls) {
```

Comment:
https://alligator.io/js/for-of-for-in-loops/

Reply:
Iterate over the requestUrls, and for each url of requestUrls, fetch that url and console.log the http status.

Reply:
👍